ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -522,152 +476,130 @@ def check_backends(logger):
-                    total_objects += objects
-                    applicable_dead_osds = bucket_safety['applicable_dead_osds']
-                    # Amount of lost disks at this point
-                    bucket[2] = bucket[2] - applicable_dead_osds
-                    disk_lost = bucket[2] - (bucket[0] + bucket[1])
-                    if disk_lost not in disk_lost_overview:
-                        disk_lost_overview[disk_lost] = 0
-                    disk_lost_overview[disk_lost] += objects
-            for disk_lost, objects in disk_lost_overview.iteritems():
-                lost = {
-                    'measurement': 'disk_lost',
-                    'tags': {
-                        'backend_name': alba_backend.name,
-                        'disk_lost': disk_lost
-                    },
-                    'fields': {
-                        'total_objects': total_objects,
-                        'objects': objects
-                    }
-                }
-                points.append(lost)
-
-        if len(points) == 0:
-            logger.skip('Found no objects present of the system.', test_name)
-        else:
-            backends_to_be_repaired = {}
-            for result in points:
-                # Ignore fully healthy ones
-                if result["tags"]["disk_lost"] == 0:
-                    continue
-                total_objects = result["fields"]["total_objects"]
-                objects = result["fields"]["objects"]
-
-                backend_name = result["tags"]["backend_name"]
-
-                if backend_name not in backends_to_be_repaired:
-                    backends_to_be_repaired[backend_name] = {"objects": 0, "total_objects": total_objects}
-                backends_to_be_repaired[backend_name]["objects"] += objects
+                    disk_lost = abs(bucket[2] - bucket_safety['applicable_dead_osds'] - (bucket[0] + bucket[1]))
+                    remaining_safety = bucket_safety['remaining_safety']
+                    if disk_lost not in lost_disks:
+                        lost_disks[disk_lost] = {"remaining_safety": remaining_safety, "objects_to_repair": bucket_safety['count']}
+                    details['total_objects'] += bucket_safety['count']
+                    if remaining_safety < 0:
+                        details['lost_objects'] += bucket_safety['count']
+                    if remaining_safety < details['lowest_safety']:
+                        details['lowest_safety'] = remaining_safety
+                    if disk_lost > details['most_disks_lost']:
+                        details['most_disks_lost'] = disk_lost
+
+        for backend_name, disk_safety_info in disk_lost_overview.iteritems():
+            # Get worst values first to see if the environment is in a critical state
+            lost_objects = disk_safety_info['lost_objects']
+            lowest_safety = disk_safety_info['lowest_safety']
+            objects_no_safety = disk_safety_info.get(0, 0)
```

Comment:
This will always be 0 since ```disk_safety_info``` will never have a key 0
```disk_safety_info['lost_disks']``` however could potentially have a key 0